src/content/articles/narrowing-function-parameters-with-rests-and-tuples.mdx

```diff
@@ -144,7 +144,7 @@ Explicit `as` assertions are still needed to access fruit-specific properties.
 TypeScript isn't able to narrow the type of `info` even though the two overloads explicitly stated that each fruit string matched up with a specific interface.
 The function's implementation signature -which is what its internal implementation respects- didn't understand the relationship.
 
-## Using Generics
+## Using Generic Functions
```

Comment:
Unrelated fixup: I've been switching my terminology over to using "generics" only as an adjective (thanks Matt Pocock!)